velox/common/caching/SsdFile.cpp

```diff
@@ -388,6 +394,7 @@ void SsdFile::write(std::vector<CachePin>& pins) {
     auto rc = folly::pwritev(fd_, iovecs.data(), iovecs.size(), offset);
     if (rc != bytes) {
       LOG(ERROR) << "Failed to write to SSD " << errno;
+      ++stats_.writeSsdErrors;
```

Comment:
```
s/writeSsdErrors/writeErrors/
```

Reply:
We have read ssd files and read checkpoint files. Similarly we have write ssd files and write checkpoint files. I would like to keep the 'ssd' or 'checkpoint' to tell them apart. Otherwise it might look confusing.
https://github.com/facebookincubator/velox/blob/ab9e260fe7a15148e24a29f8900b79de27b792ee/velox/common/caching/SsdFile.h#L157-L165